### PR TITLE
feature (data-table): new group-function.component, removed inputs from data table

### PR DESCRIPTION
**Pull request by pweyrich**
@plentymarkets/team-terra

### Definition of Done
Documentation
- [x] Changelog
- [x] Example (updated or created)
- [x] JSDoc

Testing
- [x] Unit Test (updated or created)
- [x] Person 1
- [ ] Person 2

Browser-Support
- [x] Chrome
- [x] Firefox
- [ ] Safari

### CODE REVIEW

File: src/app/components/tables/data-table/group-function/terra-group-function.component.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+        <terra-button inputIcon="icon-execute"
+                      [inputIsDisabled]="disableExecution"
+                      (outputClicked)="executeGroupFunction.emit()">
+        </terra-button>
+        <span>{{'terraDataTable.groupFunction' | translate:lang}}</span>
+    </div>
+    <div class="group-function-body">
+        <ng-content></ng-content>
+    </div>
+</div>
```
Comment (MFrank84) [inline src/app/components/tables/data-table/group-function/terra-group-function.component.html]:
```suggestion
</div>

```

File: src/app/components/tables/data-table/group-function/terra-group-function.component.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                'margin-bottom': '0'
+            })),
+            state('collapsed', style({
+                height:          '*',
+                overflow:        'initial',
+                'margin-bottom': '6px'
+            })),
+            transition('hidden <=> collapsed', [
+                animate(300)
+
```
Comment (MFrank84) [inline src/app/components/tables/data-table/group-function/terra-group-function.component.ts]:
```suggestion
```

File: src/app/components/tables/data-table/group-function/terra-group-function.component.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<div class="group-function" [@collapsedState]="collapsedState">
+    <div class="group-function-head">
+        <terra-button inputIcon="icon-execute"
+                      [inputIsDisabled]="disableExecution"
+                      (outputClicked)="executeGroupFunction.emit()">
+        </terra-button>
+        <span>{{'terraDataTable.groupFunction' | translate:lang}}</span>
```
Comment (MFrank84) [inline src/app/components/tables/data-table/group-function/terra-group-function.component.html]: i18n missing

File: src/app/components/tables/data-table/group-function/terra-group-function.component.scss [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+            vertical-align: middle;
+        }
+    }
+    
+    .group-function-body
+    {
+        padding: var(--space-lg);
+        transition: padding var(--transition-lg);
+    }
+}
```
Comment (MFrank84) [inline src/app/components/tables/data-table/group-function/terra-group-function.component.scss]:
```suggestion
}

```

**Comment by MFrank84** [comment]
The slide-in and slide-out animation does not work in Safari

File: src/app/components/tables/data-table/group-function/terra-group-function.component.html (hunk 3 above)
Comment (pweyrich) [reply src/app/components/tables/data-table/group-function/terra-group-function.component.html]: it does not work locally

**Comment by pweyrich** [comment]
> The slide-in and slide-out animation does not work in Safari
Any suggestions to solve this problem?

**Comment by MFrank84** [comment]
@pweyrich #710

**Pull request merged**